### DISCUSSION TITLE
Download failed on a special site

### DISCUSSION THREAD

**Discussion by PatMoonB** [aclap-dev/video-downloadhelper#608] [Help and Q&A]
### Your question

Hi,  I pasted the failure details below. I'm using Version 9 sice this week.
Special site is https://www.ardmediathek.de/video/nuhr-im-ersten/nuhr-im-ersten-vom-23-mai-2024/rbb-fernsehen/Y3JpZDovL3JiYl9kMDk0MzVmOS1kODljLTRlOGMtYTg5MS0zZThiYjI2NWIwMjFfcHVibGljYXRpb24
This is a german site ardmediathek.de from which I downloaded several other themes successfully
But this file fails after several minutes more than twice (and counting)
Thx for a solution
PatMoonB

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://www.ardmediathek.de/video/nuhr-im-ersten/nuhr-im-ersten-vom-23-mai-2024/rbb-fernsehen/Y3JpZDovL3JiYl9kMDk0MzVmOS1kODljLTRlOGMtYTg5MS0zZThiYjI2NWIwMjFfcHVibGljYXRpb24",
  "error": {
    "details": "SideDownload error: : [https @ 0000020eb93dbf40] Stream ends prematurely at 4702032, should be 5357624\r\n[aac_adtstoasc @ 0000020eb97865c0] Error parsing ADTS frame header!\r\n[mp4 @ 0000020eb97bde40] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input\r\n[aost#0:1/copy @ 0000020eb9932580] Error submitting a packet to the muxer: Invalid data found when processing input\r\n[out#0/mp4 @ 0000020eb97bb900] Error muxing a packet\r\n",
    "downloadable_id": "downloadable_858035793379792",
    "error": "coapp_failure",
    "id": "coapp_failure_c21964ca-e484-405e-9d83-9c59a5889eb7"
  }
}

**Comment by paulrouget** [comment]
Thanks for the report!

Note: ff failure, another `Error parsing ADTS frame header` issue.

**Comment by GrampaWildWilly** [comment]
With Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.22a1 beta, CoApp 2.0.19, VDH took 1:01 to download an MP4, size 1.2G, duration 44:57, resolution 1920x1080.  It played fine in VLC.  Since I don't speak German, the show was lost on me.  It appeared to be a showcase for standup comics, but I couldn't laugh along with the audience.  I just skimmed it & it appeared to be there, video & audio, right to the end.

By the way, the progress display on this one included the usual percent complete.  I don't know what the deal is with that other one I mentioned in #605.
